api/gasstattion.go

```diff
@@ -10,8 +10,6 @@ import (
 	"math/big"
```

Comment:
Shall me move it to a separate package? API is not pure API serving, but some business logic included now.